cli/lib/localization.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const toml = require('@iarna/toml');
+const osLocale = require('os-locale');
+
+module.exports = translate;
+
+let translations = null;
```

Comment:
Could these two variables introduce a (albeit small) memory leak? I'm thinking particularly of what might happen if there's a large test suite later on.

Reply:
I really don't think so... 🤔  They are created once< overwritten once and are not changing while the process is running, don't see where they would start leaking